fe/fe-core/src/main/java/org/apache/doris/alter/AlterHandler.java

```diff
@@ -140,7 +142,20 @@ public Long getAlterJobV2Num(org.apache.doris.alter.AlterJobV2.JobState state, l
     }
 
     public Long getAlterJobV2Num(org.apache.doris.alter.AlterJobV2.JobState state) {
-        return alterJobsV2.values().stream().filter(e -> e.getJobState() == state).count();
+        Long counter = 0L;
+
+        for (AlterJobV2 job : alterJobsV2.values()) {
```

Comment:
This method is only called in `show proc` stmt, which already check the ADMIN priv.
So no need to check priv here again.
You can just leave a comment to here for other developer to notice that.

Reply:
fixed

Reply:
fixed